src/Pages/ItemDetail/Components/Board.js

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from "react";
+import "./Board.scss";
+
+const boardContent = [1, 2, 3, 4, 5, 6, 7, 8];
+const paging = ["<", 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, ">"];
+class Board extends Component {
```

Comment:
가독성을 위해 한 줄 띄어주세요! 
더불어 전역에서 사용되는 변하지 않는 상수의 경우, 
상수명을 대문자로 작성해주는 것이 일반적인 컨벤션입니다 :) 
```suggestion
const PAGES = ["<", 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, ">"];

class Board extends Component {
```

Reply:
수정했습니다!